airswap/airswap-marketplace
# indexer store

# Description

Fixes #83

# Steps to test the change

1. Checkout this branch, make sure to have the correct version of airswap modules installed
2. Hardcode indexerSlice when initialization is fulfilled as following:
     ```
     builder.addCase(initialize.fulfilled, (state, action: PayloadAction<string[]>): IndexerState => ({
          ...state,
          isInitialized: true,
          urls: [...action.payload, 'http://localhost:4001'],
     }));
     ```
3. Start indexer node locally on orderFileter branch
4. List an NFT
5. The listed NFT should appear in collection page

# Checklist:
